bot/exts/code_jams/_views.py

```diff
@@ -134,3 +135,118 @@ async def interaction_check(self, interaction: discord.Interaction) -> bool:
             )
             return False
         return True
+
+
+class AddNoteModal(discord.ui.Modal, title="Add a Note for a Code Jam Participant"):
+    """A simple modal to add a note to a Jam participant."""
+
+    def __init__(self, member: discord.Member, mgmt_client: APIClient):
+        super().__init__()
+        self.member = member
+        self.mgmt_client = mgmt_client
+
+    note = discord.ui.TextInput(
+        label="Note",
+        placeholder="Your note..."
+    )
+
+    async def on_submit(self, interaction: discord.Interaction) -> None:
+        """Discord.py default to handle modal submission."""
+        try:
+            user = await self.mgmt_client.get(f"users/{quote_url(str(self.member.id))}/current_team",
+                                              raise_for_status=True)
+        except ResponseCodeError as err:
+            if err.response.status == 404:
+                await interaction.response.send_message(":x: Something went wrong, the user could not be found!")
+            else:
+                await interaction.response.send_message(":x: Something went wrong, we have notified the team!")
+                log.error(f"Something went wrong: {err}")
+            return
+        else:
+            jam_id = user["team"]["jam_id"]
+            try:
+                await self.mgmt_client.post(
+                    "infractions",
+                    json={
+                        "user_id": self.member.id,
+                        "jam_id": jam_id, "reason": self.note.value,
+                        "infraction_type": "note"
+                    },
+                    raise_for_status=True
+                )
+            except ResponseCodeError as err:
+                if err.response.status == 404:
+                    await interaction.response.send_message(
+                        "Something went wrong! The user could not be found!",
+                        ephemeral=True
+                    )
+                else:
+                    await interaction.response.send_message(":x: Something went wrong, we have notified the team!")
+                    log.error(f"Something went wrong: {err}")
+                return
+            else:
+                await interaction.response.send_message('Your note has been saved!', ephemeral=True)
+
+    async def on_error(self, error: Exception, interaction: discord.Interaction) -> None:
+        """Discord.py default to handle modal error."""
+        await interaction.response.send_message('Something went wrong with processing your form', ephemeral=True)
+
+
+class JamInfoView(discord.ui.View):
+    """
+    A basic view that displays basic information about a CJ participant.
+
+    Additionally, you can either view the notes that was added to the participant
+    or create one of them.
```

Comment:
Imperative mood. Also "notes that was added" should've been "notes that were added".
```suggestion
    Additionally, notes for a participant can be added and viewed.
```